packages/client/tests/functional/metrics/enabled/tests.ts

```diff
@@ -212,328 +213,341 @@ testMatrix.setupTestSuite(
-        expect(countersBefore.sort((a, b) => a.key.localeCompare(b.key))).toMatchObject([
-          {
-            key: 'prisma_client_queries_total',
-            labels: {},
-            value: 0,
-            description: 'The total number of Prisma Client queries executed',
-          },
-          {
-            key: 'prisma_datasource_queries_total',
-            labels: {},
-            value: 0,
-            description: 'The total number of datasource queries executed',
-          },
-          {
-            key: 'prisma_pool_connections_closed_total',
-            labels: {},
-            value: 0,
-            description: 'The total number of pool connections closed',
-          },
-          {
-            key: 'prisma_pool_connections_opened_total',
-            labels: {},
-            value: 1,
-            description: 'The total number of pool connections opened',
-          },
-        ])
-        expect(gaugesBefore.sort((a, b) => a.key.localeCompare(b.key))).toMatchObject([
-          {
-            key: 'prisma_client_queries_active',
-            labels: {},
-            value: 0,
-            description: 'The number of currently active Prisma Client queries',
-          },
-          {
-            key: 'prisma_client_queries_wait',
-            labels: {},
-            value: 0,
-            description: 'The number of datasource queries currently waiting for an free connection',
-          },
-          {
-            key: 'prisma_pool_connections_busy',
-            labels: {},
-            value: 0,
-            description: 'The number of pool connections currently executing datasource queries',
-          },
-          {
-            key: 'prisma_pool_connections_idle',
-            labels: {},
-            value: expect.any(Number),
-            description: 'The number of pool connections that are not busy running a query',
-          },
-          {
-            key: 'prisma_pool_connections_open',
-            labels: {},
-            value: 1,
-            description: 'The number of pool connections currently open',
-          },
-        ])
-        expect(histogramsBefore.sort((a, b) => a.key.localeCompare(b.key))).toMatchObject([
-          {
-            key: 'prisma_client_queries_wait_histogram_ms',
-            labels: {},
-            value: {
-              buckets: [
-                [0, 0],
-                [1, 1],
-                [5, 0],
-                [10, 0],
-                [50, 0],
-                [100, 0],
-                [500, 0],
-                [1000, 0],
-                [5000, 0],
-                [50000, 0],
-              ],
-              sum: expect.any(Number),
-              count: 1,
+      // TODO test fails with Expected `"value": 1"` but got `"value": 0` for key "prisma_pool_connections_opened_total" See https://github.com/prisma/team-orm/issues/372
+      skipTestIf(providerFlavor === ProviderFlavors.JS_NEON || providerFlavor === ProviderFlavors.JS_PG)(
```

Comment:
This is expected for now as `prisma_pool_connections_opened_total` comes from mobc

Reply:
Expecting 0 for the driver adapters is definitely ok for now.